direct/client.go

```diff
@@ -43,41 +43,18 @@ func (c *directClient) Entry(ctx context.Context, pubKey cipher.PubKey) (*disc.E
 }
 
 // PostEntry adds a new Entry.
-func (c *directClient) PostEntry(ctx context.Context, e *disc.Entry) error {
+func (c *directClient) PostEntry(ctx context.Context, entry *disc.Entry) error {
```

Comment:
I'm using the [APIClient](https://github.com/ersonp/dmsg/blob/5eea5d6365c70c3f7e05b7bcfd1dd56eeffa0add/disc/client.go#L22) interface for it so that it will be used as a memory version of the `disc`.

Reply:
One way to fix the naming issue will be to change the func names in [APIClient interface](https://github.com/ersonp/dmsg/blob/5eea5d6365c70c3f7e05b7bcfd1dd56eeffa0add/disc/client.go#L22) to 
```
PostEntry -> SaveEntry
PutEntry -> UpdateEntry
```
And use `postEntry` and `PutEntry` in the `disc` inside of `SaveEntry` and `UpdateEntry` respectively. Also the function signature will be the same as we have for `Redis` and `memory` store in services where the funcs in the `memory store` don't return any errors.